props/saber_BC_buttons.h

```diff
@@ -437,6 +437,53 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     SaberBase::DoEffect(EFFECT_FAST_ON, 0);
   }
 
+  // Go to the next Preset skipping Preon effect with FastOn.
```

Comment:
Let's move these functions to PropBase instead of duplicating it in every prop file.

Reply:
Sounds good. It IS right from Fett263_buttons.
Is that some of the last ironing-out for OS6, or did you mean for me to actually PR it into prop_base?

Reply:
I meant for you to use a PR to move it into prop_base

Reply:
> I meant for you to use a PR to move it into prop_base

Ok. So in doing so, there's a macro conflict. It compiles, but with warnings of "already defined" for SET_BLADE_STYLE. While looking for ways to sort it out, I spoke with Fett263 since it was his code I copied.
He said this ties into Edit Mode and the whole breaking-things-out complication at the moment.
Maybe duplicating in each prop for now is easiest? As long as it's not "wrong", this works great for me.

Reply:
Just rename the define, or just add #undef SET_BLADE_STYLE after it's been used.

Reply:
Renamed the define, updated prop_base in #263